src/index.html

```diff
@@ -11,12 +11,106 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          width="40"
+          height="40"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
```

Comment:
The `href` attribute for this link is set to 'index.html'. If this link is supposed to navigate to a different page, make sure to update the `href` attribute accordingly.